### PR TITLE
Dev 002

### PR DESCRIPTION
**Pull request by pkongyang**
## Rationale
Create Navbar and fix up page margin
- Search box
- logo
- mobile view

## Advice for Reviewers & Testing Notes
- Run the page locally
- Please see if it work and the color is ok
- The Navbar is fixed at the top.
- no margin
- check the folder structure is correct.

## Screenshots:

**Mobile view:**
image

**Desktop view:**
image

## Task Name

-  DEV-002 Navbar UI

## Linting Checklist
- [ ] No commented code
- [ ] Code Formatted nicely (Prettier)
- [ ] PR your own code before you assign reviewers

### CODE REVIEW

File: .gitignore [hunk 1]
```diff
@@ -0,0 +1 @@
+node_modules
```
Comment (flormcalvo) [inline .gitignore:1]: I think here is "/node_modules"

File: client/src/App.js [hunk 2]
```diff
@@ -1,13 +1,13 @@
+/** @format */
```
Comment (tangnology) [inline client/src/App.js:1]: not sure why these are appearing at the top of the files, could you please remove

File: client/src/components/Navbar/Navbar.js [hunk 3]
```diff
@@ -0,0 +1,107 @@
+/** @format */
```
Comment (tangnology) [inline client/src/components/Navbar/Navbar.js:1]: similar to other comments

File: client/src/index.js [hunk 4]
```diff
@@ -1,10 +1,14 @@
-import React from 'react';
-import ReactDOM from 'react-dom';
-import App from './App';
+/** @format */
```
Comment (tangnology) [inline client/src/index.js:1]: please remove

File: client/src/components/Navbar/Navbar.js [hunk 5]
```diff
@@ -0,0 +1,107 @@
...
+
+import React from 'react'
+import AppBar from '@material-ui/core/AppBar'
+import Toolbar from '@material-ui/core/Toolbar'
+import IconButton from '@material-ui/core/IconButton'
+import Typography from '@material-ui/core/Typography'
+import InputBase from '@material-ui/core/InputBase'
+import { fade, makeStyles } from '@material-ui/core/styles'
+import SearchIcon from '@material-ui/icons/Search'
+import Logo from './asset/logo.png'
```
Comment (tangnology) [inline client/src/components/Navbar/Navbar.js:11]: could you please move the logo image out to the static folder, we'll be saving any static files images in that folder

File: client/src/components/Navbar/Navbar.js (hunk 3 above)
Comment (tangnology) [inline client/src/components/Navbar/Navbar.js:1]: similar to DEV-015, please structure the Navbar component, like the Card component file structure

File: .gitignore (hunk 1 above)
Comment (pkongyang) [reply .gitignore:1]: yes you're right
I will delete this file

File: client/src/App.js (hunk 2 above)
Comment (pkongyang) [reply client/src/App.js:1]: I use prettier. when I save it auto came up.

File: client/src/components/Navbar/Navbar.js (hunk 3 above)
Comment (pkongyang) [reply client/src/components/Navbar/Navbar.js:1]: I use prettier. when I save it auto came up.

File: client/src/App.js (hunk 2 above)
Comment (pkongyang) [reply client/src/App.js:1]: I use prettier. when I save it auto came up.

File: client/src/components/Navbar/Navbar.js [hunk 6]
```diff
@@ -0,0 +1,107 @@
...
+
+const useStyles = makeStyles((theme) => ({
+  root: {
+    flexGrow: 1,
+    background: '#2B4C72',
+    color: 'white',
+    position: 'fixed',
+    width: '100%',
+  },
+  menuButton: {
```
Comment (tangnology) [inline client/src/components/Navbar/Navbar.js:21]: dont think this one's needed

Comment (pkongyang) [reply client/src/components/Navbar/Navbar.js:21]: this to fix Navbar on top at all time

**Pull request merged**